server.js

```diff
@@ -33,6 +33,13 @@ cors_proxy.createServer({
     'x-heroku-queue-depth',
     'x-heroku-dynos-in-use',
     'x-request-start',
+    'x-request-id',
+    'x-forwarded-for',
```

Comment:
I have no objection against the removal of the listed headers, except maybe for `x-forwarded-*` because it is a standard non-Heroku specific header, and people may be relying on its existence.